app/Main.hs

```diff
@@ -1,21 +1,14 @@
 {-# LANGUAGE QuasiQuotes #-}
 {-# LANGUAGE TemplateHaskell #-}
 
-import System.Environment (getArgs)
-import System.Exit (exitFailure)
-import System.IO (hPutStrLn, stderr)
-
-import Control.Exception.Safe
 import Control.Applicative (optional)
 import Data.Semigroup ((<>))
-import Data.Text (unpack)
 import qualified Options.Applicative as Opt
 import Options.Applicative ((<**>), many)
-import Control.Monad.Reader
-import PyF
+
+import System.Console.Pretty
```

Comment:
One more library for my toolbox. I had not looked for it. I did `githud` with my little fingers and hidden console control characters :/

(well maybe I had to anyway since I touch the prompt, but....)

Cool

Reply:
Do we have standards on limited/qualified import.
Last time we discussed, I thought we agreed that we liked to know where functions/types were coming from and that the import therefore should be explicitly listed or qualified

Reply:
Do we have standards on limited/qualified import.
Last time we discussed, I thought we agreed that we liked to know where functions/types were coming from and that the import therefore should be explicitly listed or qualified

Reply:
I totally agree, especially considering that I'm the author of a, now refused, GHC proposal about qualified imports. I'll do a fix for that.